app/services/job_posting_api/poller.rb

```diff
@@ -56,6 +56,8 @@ def process_feed(feed)
 
     def sync_posting(attributes)
       job_posting = JobPosting.find_or_initialize_by(identification_number: attributes[:identification_number])
+      return unless job_posting.scraped?
```

Comment:
Currently, if an organization claimed its profile, it would've been updated in the next polling round